db/migrate/20251107131457_add_record_deleted_to_ocdc_tables.rb

```diff
@@ -0,0 +1,19 @@
+class AddRecordDeletedToOcdcTables < ActiveRecord::Migration[7.0]
+  def up
```

Comment:
These fields exist in the database, but were not in the structure.sql file or migration history